### PR TITLE
bugfix: remove warning in unittests

### PR DESCRIPTION
**Pull request by Mcthomas777**
In process, just using pipeline for testing purpose

### CODE REVIEW

**Comment by Mcthomas777** [comment]
Looks like working fine, with no warning !

**Review by sebastianpfischer — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
So it is definitely better but:
```

=============================== warnings summary ===============================
tests/test_cc_uart.py::test_virtual_communication
  /home/runner/.local/share/virtualenvs/kiso-testing-4OSmd96j/lib/python3.8/site-packages/_pytest/threadexception.py:73: PytestUnhandledThreadExceptionWarning: Exception in thread Thread-63
  
  Traceback (most recent call last):
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 932, in _bootstrap_inner
      self.run()
    File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/threading.py", line 870, in run
      self._target(*self._args, **self._kwargs)
    File "/home/runner/work/kiso-testing/kiso-testing/tests/test_cc_uart.py", line 54, in _communication_send_check
      message_to_send = message.Message(
  TypeError: __init__() got an unexpected keyword argument 'test_section'
  
    warnings.warn(pytest.PytestUnhandledThreadExceptionWarning(msg))

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

---------- coverage: platform linux, python 3.8.12-final-0 -----------
Coverage HTML written to dir ./tests/coverage_report.html
Coverage XML written to file coverage.xml

================== 526 passed, 1 skipped, 1 warning in 13.86s ==================

```

**Review by sebastianpfischer — CHANGES_REQUESTED** [review]
There is one more warning left :)

**Review by sebastianpfischer — APPROVED** [review]

File: Pipfile [hunk 1]
```diff
@@ -31,3 +31,4 @@ robotframework = "3.2.2"
 pyvisa = "*"
 pyvisa-py = "*"
 python-uds = {ref = "2.0.2", git = "https://github.com/BKaDamien/python-uds"}
+pre-commit = "*"
```
Comment (sebclrsn) [inline Pipfile:34]: Should be part of the dev-packages

**Review by sebclrsn — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ repos:
       - id: isort
         args: ["--profile", "black", "--line-length", "79"]
 -   repo: https://github.com/psf/black
-    rev: 21.11b1
+    rev: 22.3.0
```
Comment (Pog3k) [inline .pre-commit-config.yaml:8]: Is there a reason for the update ? 

Now you may break the format check because it is not updated in the pipfile

**Review by Pog3k — CHANGES_REQUESTED** [review]
Fix Black in Pipfile or Revert version in pre commit

File: .pre-commit-config.yaml (hunk 2 above)
Comment (Mcthomas777) [reply .pre-commit-config.yaml:8]: I just had some trouble locally with this version, as dependency  click (afair) has changed a bit, so I update it.
I just changed it into the pipfile

Comment (sebastianpfischer) [reply .pre-commit-config.yaml:8]: @Pog3k , we all had. There is some issue between black and the latest click. This make all pre-commit fail.

Comment (Pog3k) [reply .pre-commit-config.yaml:8]: @Mcthomas777 , @sebastianpfischer 
Thx for the info.
Then update it in the pipfile..

**Review by kakooloukia — APPROVED** [review]

**Review by sebclrsn — APPROVED** [review]

**Pull request merged**